src/analysis/souffle/sql_verifier_interface.dl

```diff
@@ -0,0 +1,32 @@
+//-----------------------------------------------------------------------------
```

Comment:
I think it is OK to have a separate `sql_verifier_interface.dl` for now, but ideally we should just have something like `policy_verifier_interface.dl`?

Reply:
Ideally, yes. For now, however, we have to deal with the special PolicyRules that the SQL Verifier is giving us, which has to be consumed specially as a different `.input` relation. That requires a different interface than a general policy verifier.

I think it's possible to do something more general in the future and fold this into a more general policy verifier library. For instance, you pointed out in one of the meetings recently that the "policy rules" generated by the SQL verifier are more like implementations for overlay `Operation`s. If we can refactor policy rules to fit into a more general notion of `Operation` models, then we no longer need a special interface. For now, however, we do.

Reply:
Renamed this to policy_verifier_interface.dl in recognition of the fact that we want this interface to be the generic Datalog policy verification interface, even if it has a few SQL-specific items in it at the moment.